Lib/importlib/_bootstrap_external.py

```diff
@@ -360,6 +360,7 @@ def _write_atomic(path, data, mode=0o666):
 #     Python 3.11a1 3455 (add MAKE_CELL bpo-43693)
 #     Python 3.11a1 3456 (interleave cell args bpo-43693)
 #     Python 3.11a1 3457 (Change localsplus to a bytes object bpo-43693)
+#     Python 3.11a1 3458 (imported objects now don't use LOAD_METHOD/CALL_METHOD)
```

Comment:
Shouldn't the actual number be bumped below? I don't think it matters much in this case, but for the sake of consistency/clarity for the next bump, it might be good to fix.

Reply:
Right, probably got away during rebase. Will fix